workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/pages/AllScreens.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayLayout from '@clayui/layout';
+import {ClayVerticalNav} from '@clayui/nav';
+import {ReactNode} from 'react';
+
+import {useHash} from '../services/useHash';
+import {ScreenType} from '../types';
+import TicketsDashboardApp from './TicketsDashboard';
+import TicketApp from './TicketsOverview';
+
+const routes = [
+	{
+		element: <TicketsDashboardApp screenType={ScreenType.INTEGRATED} />,
```

Comment:
TicketsDashboardApp needs the same name change that was done in the other files.